OrbitQt/ProcessItemModel.h

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_QT_PROCESS_ITEM_MODEL_H_
+#define ORBIT_QT_PROCESS_ITEM_MODEL_H_
+
+#include <QAbstractItemModel>
+#include <QVariant>
+
+#include "process.pb.h"
+
+namespace OrbitQt {
+
+class ProcessItemModel : public QAbstractItemModel {
+  // Q_OBJECT
+ public:
+  enum class Column { kName, kPid, kCpu, kEnd };
+
+  int columnCount(const QModelIndex& parent = {}) const override;
+  QVariant data(const QModelIndex& idx, int role = Qt::DisplayRole) const override;
+  // maybe flags?
```

Comment:
Is this comment intended?

Reply:
no, just a mistakes on my side. I should read over changes one final time before I create the PR. :/